### PR TITLE
ST: Add some updates to current tests make them working with OLM

### PR DESCRIPTION
**Pull request by Frawless**
Signed-off-by: Jakub Stejskal <xstejs24@gmail.com>

### Type of change

- Bugfix

### Description
This PR adds several bugfixes for our tests. The problems occurred when we run the tests with CO installed via OLM.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java [hunk 1]
```diff
@@ -148,6 +149,7 @@ protected void installClusterOperator(String namespace) throws Exception {
      * @param resources list of path to yaml files with resources specifications
      */
     protected void prepareEnvForOperator(String clientNamespace, List<String> namespaces, String... resources) {
+        assumeTrue(!Environment.isHelmInstall() && !Environment.isOlmInstall());
```
Comment (see-quick) [inline systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java:152]: Hmmm, don't we use `asserThat(condition, is(true));` on everything?

**Review by see-quick — APPROVED** [review]
One commends and no need to review it again :)

File: systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java (hunk 1 above)
Comment (Frawless) [reply systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java:152]: assume != assert

**Review by scholzj — APPROVED** [review]

**Pull request merged**